src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -562,13 +593,23 @@ def BuildDpa(dpa_name, protection_points_method=None):
     IOError: if the provided file cannot be found.
     ValueError: if the provided file is not a valid GeoJSON MultiPoint geometry.
   """
+  try:
+    dpa_zone = zones.GetCoastalDpaZones()[dpa_name]
```

Comment:
Not important. But in the "Coastal_DPAs.kml", shouldn't <Data name="portalOrg"> be <Data name="coastalOrg">?

Reply:
I dont think so. portalOrg is only meaningful for the Portal-DPA, and not for the Coastal-DPA KML.
The fact that it is there in the Coastal is just for convenience by Nick I guess. The software will ignore it on the Coastal DPA KML.
From Andy Clegg proposed document describing the format:

> For P-DPAs, the name of an informing incumbent portal organization to which the DPA belongs. Multiple organizations can be listed. This establishes the authorization for portal users under the listed organization(s) to be allowed to “book” conference rooms (channels) belonging to this building (DPA). If no organizations are listed, only portal administrators will be able to activate any channels in this DPA.